tutorials/high_performance_networking/README.md

```diff
@@ -0,0 +1,2171 @@
+16. A descriptive name for that queue, currently only used for logging.
+17. The ID of that queue, which can be referred to later in the `flows` section.
+18. The number of packets per batch. The `advanced_network` Rx and Tx operators will "tick" for each batch.
+19. The ID of the CPU core that this queue will use to poll the NIC. Ideally one [isolated core](#35-isolate-cpu-cores) per queue.
+20. The list of memory regions where this queue will write/read packets from/to. The order matters: the first memory region will be used first to read/write from until it fills up one buffer (`buf_size`), after which it will move to the next region in the list and so on until the packet is fully written/read. See the `memory_regions` for the `rx` queue below for an example.
+21. Same as for `data1`. Each interface in this list should have a unique mac address. This one will do `rx` per config below.
+22. Whether to isolate the Rx flow. If true, any incoming packets that does not match the MAC address of this interface - or isn't directed to a queue when the `flows` section below is used - will be delegated back to Linux for processing (no kernel bypass). This is useful to let this interface handle ARP, ICMP, etc. Otherwise, any packets sent to this interface (ex: ping) will need to be processed (or dropped) by your application.
+23. `rx` queues have an `output_port` parameter so you can attach a downstream operator to receive data from this specific queue, as can be seen in the `#!cpp Application::compose()` function of the sample application. Multiple `rx` queues can share the same `output_port`. In contrast, `tx` queues have a single non-configurable port (name: `burst_in`) to which upstream operators will send all packets, which are then routed to the correct queue based on #TODO#.
+24. This scenario is called HDS (Header-Data Split): the packet will first be written to a buffer in the `Data_RX_CPU` memory region, filling its `buf_size` of 64 bytes - which is consistent with the size of our header - then the rest of the packet will be written to the `Data_RX_GPU` memory region. Its `buf_size` of 1000 bytes is just what we need to write the payload size for our application, no byte wasted!
+25. The list of flows. Flows are responsible for routing packets to the correct queue based on various properties. If this field is missing, all packets will be routed to the first queue.
+26. The flow name, currently only used for logging.
+27. The flow `id` is used for #TODO#
```

Comment:
@cliffburdick help please

Reply:
The flow ID is used to tag the packets with what flow it arrived on. This is useful when sending multiple flows to a single queue, and the user application can differentiate which flow matched the packet based on this ID.

Reply:
great